internal/pkg/cli/svc_init.go

```diff
@@ -604,24 +621,45 @@ func (o *initSvcOpts) askSvcPort() (err error) {
 		return nil
 	}
 
-	port, err := o.prompt.Get(
-		fmt.Sprintf(svcInitSvcPortPrompt, color.Emphasize("port")),
+	if o.wkldType == manifestinfo.RequestDrivenWebServiceType {
+		selectedPorts, err := o.prompt.Get(
+			fmt.Sprintf(svcInitSvcPortPrompt, color.Emphasize("port")),
+			svcInitSvcPortHelpPrompt,
+			validateSvcPort,
+			prompt.WithDefaultInput(defaultPort),
+			prompt.WithFinalMessage("Port:"),
+		)
+		if err != nil {
+			return fmt.Errorf("get port: %w", err)
+		}
+		if len(strings.Split(selectedPorts, ",")) > 1 {
+			return fmt.Errorf("App Runner Service doesn't expose multiple ports")
```

Comment:
I'm confused about the error case here: is this saying that you can't expose multiple ports with a RDWS?